client/src/components/Drug/DrugRecord/DrugRecord.scss

```diff
@@ -1,6 +1,26 @@
 .drug-record-container {
   padding: 20px;
 
+  .drug-record-header {
+    display: flex;
+    flex-direction: row;
+    align-items: end; // TODO ??
```

Comment:
I couldn't get margin/vertical-align to put this where I wanted, so I manually inserted some margin (line 20)